test/e2e/specs/media/media__edit.ts

```diff
@@ -19,49 +23,50 @@ declare const browser: Browser;
 describe( DataHelper.createSuiteTitle( 'Media: Edit Media' ), function () {
```

Comment:
**comment:non-blocking** - unrelated to the scope of this, but we should rewrite this spec to upload and delete the image using the RestAPIClient in order to limit the scope of the feature under test to just the editing process.

Reply:
That's a great idea!! 🎉